app/decorators/case/overturned_ico/sar_decorator.rb

```diff
@@ -4,5 +4,8 @@ def original_case_description
     "ICO appeal (SAR) #{object.original_ico_appeal.number}"
   end
 
+  def type_abbreviation
+    'Overturned SAR'
```

Comment:
Why is this in the decorator? All the other type_abbreviations are directly in the models. it also saves us calling decorate on the object.

Reply:
Theres no tests for this method

Reply:
it''s in the decorator because "Overturned SAR created" reads better in a flash message than "OVERTURNED_SAR created"